src/site/stages/build/drupal/api.js

```diff
@@ -108,6 +115,28 @@ function getDrupalClient(buildOptions) {
       });
     },
 
+    getRemainingContent(onlyPublishedContent = true) {
+      return this.query({
+        query: getQuery(queries.GET_REMAINING_CONTENT),
```

Comment:
It's not immediately obvious what this is referring to (without additional context). What do you think about `GET_NON_NODE_CONTENT` or something more descriptive?

Reply:
I'm all for a better name :smile: